packages/ui-scripts/lib/bump.js

```diff
@@ -66,5 +70,13 @@ async function bump(
     process.exit(1)
   }
 
+  info(`💾  Tagging release commit...`)
```

Comment:
I'd add here some text that the tag needs to be pushed too with e.g. `git push --follow-tags`